admin/app/components/solidus_admin/sidebar/item/component.rb

```diff
@@ -11,4 +11,30 @@ def initialize(item:)
   def name
     t("main_nav.#{@item.key}")
   end
+
+  # Arbitrary Tailwind background images is not working:
+  # https://github.com/tailwindlabs/tailwindcss/discussions/6617
+  def background_image_style_attribute
+    return unless @item.icon
+
+    "style=\"background-image: url('".html_safe +
+      image_path(@item.icon) +
+      "')\"".html_safe
```

Comment:
Good point. However, in this case, I wanted to be picky and not mark the URL as safe, in case someone implements another kind of main menu item fetching data from the database. In any case, that's not part of the final version anymore :slightly_smiling_face: